test/lit/passes/signature-refining.wast

```diff
@@ -508,16 +508,16 @@
   (type $sig-can-refine (func_subtype (result anyref) func))
 
   ;; Also a single function, but no refinement is possible.
-  ;; CHECK:      (type $sig-cannot-refine (func (result anyref)))
-  (type $sig-cannot-refine (func_subtype (result anyref) func))
+  ;; CHECK:      (type $sig-cannot-refine (func (result (ref func))))
+  (type $sig-cannot-refine (func_subtype (result funcref) func))
```

Comment:
Comment says no refinement is possible, but the type is refined from `funcref` to `ref func` :thinking:

Reply:
Ah oops, good catch.